internal/provider/data_source_test.go

```diff
@@ -255,6 +145,58 @@ func TestDataSource_utf16(t *testing.T) {
 // 	})
 // }
 
+const testDataSourceConfigBasic = `
+data "http" "http_test" {
+  url = "%s/meta_%d.txt"
+}
+
+output "body" {
+  value = data.http.http_test.body
+}
+
+output "response_headers" {
+  value = data.http.http_test.response_headers
+}
```

Comment:
Testing outputs is generally extraneous as compared to just directly checking the data source in question. 👍 

```suggestion
```

Reply:
Removed.